### PR TITLE
Fixing typos, adding commas, rewording a few things

### PR DESCRIPTION
**Pull request by T0rrn**

### CODE REVIEW

File: pl.json [hunk 1]
```diff
@@ -345,6 +345,7 @@
         "Account.ErrorEmailMismatch" : "Emaile nie są takie same!",
         "Account.ErrorLoginError" : "Błąd Logowania:",
         "Account.RegistrationSuccess" : "Udało się! Został wysłany email z linkiem potwierdzającym. Po otworzeniu linku będziesz w stanie się zalogować.",
+        "": "<size=120%>Udało się!</size><br><br>Email z linkiem potwierdzającym został wysłany do {email}.<br><br>Po otworzeniu linku, zostaniesz zalogowany automatycznie jeżeli to okienko jest otwarte we'll automatically log you in as long as this dialog is open.<br><br>Jeśli napotkasz problemy, napisz do nas na hello@neos.com lub discord.gg/neosvr",
```
Comment (art0007i) [inline pl.json]: what is this

File: pl.json [hunk 2]
```diff
@@ -298,10 +298,10 @@
         "Options.SeatedMode.On" : "Tryb Siedzący: Tak",
         "Options.SeatedMode.Off" : "Tryb Siedzący: Nie",
         "Options.Nameplates.All" : "Nazwy Graczy: Wszystkie",
-        "Options.Nameplates.NonContacts" : "Nazwy Graczy: Nie Kontakty",
+        "Options.Nameplates.NonContacts" : "Nazwy Graczy: Nie Kontaktów",
         "Options.Nameplates.Hide" : "Nazwy Graczy: Ukryte",
 
-        "Indicator.OnlineUsers" : "Użytkownicy Online: ~{num_total} ({num_registered})",
+        "Indicator.OnlineUsers" : "Użytkownicy Online: ~{num_registered} ({num_total})",
```
Comment (art0007i) [inline pl.json]: i see you tried to fix this but its  still wrong, it should be `{num_registered} (~{num_total})`

**Review by art0007i — CHANGES_REQUESTED** [review]
everything looks good except a couple things

File: pl.json (hunk 1 above)
Comment (T0rrn) [reply pl.json]: Looks like I copied the English value into an empty key to have this and previous version side-by-side (English version seems to have been updated), but didn't fix the key afterwards. Thanks for catching this

**Pull request merged**